packages/beacon-node/src/chain/validation/inclusionList.ts

```diff
@@ -1,32 +1,57 @@
 import {SLOTS_PER_EPOCH} from "@lodestar/params";
 import {computeEpochAtSlot, getInclusionListSignatureSet} from "@lodestar/state-transition";
 import {eip7805} from "@lodestar/types";
+import {Metrics} from "../../metrics/index.js";
 import {getShufflingDependentRoot} from "../../util/dependentRoot.js";
 import {InclusionListError, InclusionListErrorCode} from "../errors/inclusionList.js";
 import {GossipAction} from "../errors/index.js";
 import {IBeaconChain} from "../index.js";
 
+export enum InclusionListSource {
+  gossip = "gossip",
+  api = "api",
+}
+
+export enum InvalidInclusionListReason {
+  maxSizeExceeded = "max_size_exceeded",
+  slotNumberError = "slot_number_error",
+  ilCommitteeShuffling = "il_committee_shuffling",
+  validatorNotInCommittee = "validator_not_in_committee",
+  seenTwice = "seen_twice",
+  invalidSignature = "invalid_signature",
+  clientImplementation = "client_implementation",
```

Comment:
this error case does not make sense, for now just add `unkown` or similar but we shouldn't have those kind of error cases later